test/e2e/tests/confirmations/signatures/sign-typed-data-v4.spec.ts

```diff
@@ -111,13 +112,40 @@ describe('Confirmation Signature - Sign Typed Data V4', function (this: Suite) {
       },
     );
   });
+
+  it('signs message with verifyingContract field missing', async function () {
+    await withTransactionEnvelopeTypeFixtures(
+      this.test?.fullTitle(),
+      TransactionEnvelopeType.legacy,
+      async ({
+        driver,
+      }: TestSuiteArguments) => {
+        await unlockWallet(driver);
+        await openDapp(
+          driver,
+          null,
+          `${DAPP_URL}/request?method=eth_signTypedData_v4&params=["${DEFAULT_FIXTURE_ACCOUNT}",{"types":{"EIP712Domain":[{"name":"name","type":"string"},{"name":"chainId","type":"uint256"},{"name":"version","type":"string"}],"Person":[{"name":"name","type":"string"},{"name":"wallets","type":"address[]"}],"Mail":[{"name":"from","type":"Person"},{"name":"to","type":"Person[]"},{"name":"contents","type":"string"},{"name":"attachment","type":"bytes"}]},"primaryType":"Mail","domain":{"chainId":"0x539","name":"Ether Mail","version":"1"},"message":{"contents":"Hello, Bob!","from":{"name":"Cow","wallets":["0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826","0xDeaDbeefdEAdbeefdEadbEEFdeadbeEFdEaDbeeF"]},"to":[{"name":"Bob","wallets":["0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB","0xB0BdaBea57B0BDABeA57b0bdABEA57b0BDabEa57","0xB0B0b0b0b0b0B000000000000000000000000000"]}],"attachment":"0x"}}]`,
```

Comment:
:information_source:  using the same test dapp message for v4 without the verifying contract field. This way, we can use the `assertInfoValues` method almost identically , with the caveat that now there's not verifying contract field